guacamole-common/src/main/java/org/apache/guacamole/protocol/GuacamoleClientInformation.java

```diff
@@ -144,5 +149,30 @@ public void setOptimalResolution(int optimalResolution) {
     public List<String> getImageMimetypes() {
         return imageMimetypes;
     }
+    
+    /**
+     * Return the timezone as reported by the client, or null if the timezone
+     * is not set.  Valid timezones are specified in tz database format.
```

Comment:
The JSTZ documentation states:

> The return value is an IANA zone info key (aka the Olson time zone database).
>
> The IANA timezone database is pretty much standard for most platforms (UNIX and Mac support it natively, and every programming language in the world either has native support or well maintained libraries that support it).

Is the correct term for this "tz database format"? Or "IANA timezone database format"? Or "IANA zone info key"? Or ... ?

Reply:
Possibly relevant: https://www.iana.org/time-zones

Reply:
Ah, okay.  I found it, here:
https://en.wikipedia.org/wiki/Tz_database

Reply:
Updated this a bit.